masterslave.q

```diff
@@ -0,0 +1,62 @@
+//Master slave algorithm
+//Each process initially assumes it's the master process
+//Master process is determined by oldest process start time 
+//Connect to other slave processes and retrieve details to determine which process is master
+
+//run lines
+//q torq.q -load masterslave.q -proctype lb -procname lb1 -p 6100 -debug -parentproctype wdb
+//q torq.q -load masterslave.q -proctype lb -procname lb2 -p 6101 -debug -parentproctype wdb
+
+
+// the list of processes to connect to
+.servers.CONNECTIONS:`lb
+
+// custom function, this is invoked when a new outbound connection is created
+// to be customised to invoke negotiation of processes
+.servers.connectcustom:{.lg.o[`connect;"found new connection"]; show x}
+
+// create connections
+.servers.startup[]
+
+\d .masterslave
+
+//table scehma of connected lb processes with ismaster status
+statustable:([handle:`int$()] procname:`symbol$();starttimeUTC:`timestamp$();ismaster:`boolean$())
+
+//store own process start timestamp
+start:.z.p
+
+//set details dict with own details
+details:{`procname`starttimeUTC`ismaster!(.proc.procname;.masterslave.start;1b)} 
```

Comment:
I think probably for now we can drop the ismaster value
We can just pass the name and the start time, and then both sides of the negotiation can work out which is the master (which might be neither of them)